### ISSUE TITLE
Adding bitmap text does not work

### ISSUE THREAD

**Issue by Kababaer** [PhaserEditor2D/PhaserEditor#112]
## Version

* Phaser Editor Version: 1.5.3

* Operating system: Win10

## Description

Within a Group Prefab, right click in the canvas editor -> Text -> Add Bitmap Text -> Enter a random text -> choosing a bitmap font from the menu which is showing -> hit ok -> confirm, that nothing happens.
It's not added to the canvas nor to the left "asset window".
The bitmap file is *.fnt.

Edit: The bitmap was created using littera. I just see, when I drop the bitmap font into the preview, it says: Missing bitmap font descriptor.

**Comment by PhaserEditor2D** [comment]
Hi, please attach here the fnt file. Phaser only supports Xml and Json.

**Comment by Kababaer** [comment]
Cannot attach that file typ. I don't wanna write the complete file here as it has more than hundred lines.
Littera offered the file typ XML (*fnt) but only gives me the *.fnt and the *.png.

Can you suggest a workaround?

**Comment by PhaserEditor2D** [comment]
Hi, you can find here the tools Phase

**Comment by PhaserEditor2D** [comment]
By the way, if littera exports to XML, try with it, no matter if the file extension if `fnt`.

**Comment by Kababaer** [comment]
The unfunny thing is that littera offers the xml export but just give the *.fnt which actually looks like *.xml as it has a lot of tags and attributes like <char xSpacing=""..../char>. However, this is information seems to be "not enough" for phaser editor as something seems to be missing. The bitmap font is working ingame, but not in preview. The idea was to design a complete dialoge as group prefab inclusive the text and make the text wrapping in the prefab too, if necessary (instead of calculate all that).

**Comment by PhaserEditor2D** [comment]
Please, just paste a few lines of the content of the xml/fnt file here. Other users had issues with the format of the xml.

**Comment by Kababaer** [comment]
Sry, wasn't here a long time ago.

```
<font>
  <info face="font" size="72" bold="1" italic="0" charset="" unicode="" stretchH="100" smooth="1" aa="1" padding="2,2,2,2" spacing="0,0" outline="0"/>
  <common lineHeight="72" base="63" scaleW="694" scaleH="511" pages="1" packed="0"/>
  <pages>
    <page id="0" file="font.png"/>
  </pages>
  <chars count="93">
    <char id="97" x="2" y="2" width="56" height="47" xoffset="9" yoffset="18" xadvance="72" page="0" chnl="15"/>
    <char id="98" x="2" y="51" width="56" height="65" xoffset="9" yoffset="0" xadvance="72" page="0" chnl="15"/>
    <char id="99" x="2" y="118" width="56" height="47" xoffset="9" yoffset="18" xadvance="72" page="0" chnl="15"/>
    <char id="100" x="2" y="167" width="56" height="65" xoffset="9" yoffset="0" xadvance="72" page="0" chnl="15"/>
    <char id="101" x="2" y="234" width="56" height="47" xoffset="9" yoffset="18" xadvance="72" page="0" chnl="15"/>
    <char id="102" x="2" y="283" width="56" height="65" xoffset="9" yoffset="0" xadvance="63" page="0" chnl="15"/>
    <char id="103" x="2" y="350" width="56" height="56" xoffset="9" yoffset="18" xadvance="72" page="0" chnl="15"/>
  </chars>
</font>
```

Within the chars-tag, there a many more chars. Just excluded them to keep the lines of xml small.

**Comment by PhaserEditor2D** [comment]
Well, I don't see any problem with the format. If you can send me the whole file and texture I can test here I find the problem.

**Comment by Kababaer** [comment]
Sent you guys an e-mail due to upload restrictions.

**Comment by PhaserEditor2D** [comment]
Hi,

Thanks for sending me the files.

It is working to me! The trick is to rename the file `Pixel8Active.fnt` to `assets/Pixel8Active.fnt.xml`, you should add a `.xml` extension. Please, try and tell me if it works for you.

![image](https://user-images.githubusercontent.com/12187299/55232902-418f5900-51fd-11e9-8428-4aafdbff8b7d.png)

**Comment by PhaserEditor2D** [comment]
By the way, I recommend to use the latest version of the v1 series: `1.5.3`.

**Comment by PhaserEditor2D** [comment]
Can I close this?

**Comment by Kababaer** [comment]
Sry, had vacation. Will test it out tomorrow and write.

**Comment by PhaserEditor2D** [comment]
@Kababaer there is no hurry! Enjoy the vacations!

**Comment by PhaserEditor2D** [comment]
Done, will be included in 1.4.5 very soon.
